Converted mob_item_ratio to YAML (#6021)
rathena/rathena: doc/mob_item_ratio.txt

//===== rAthena Documentation ================================
//= Mob Item Ratio Database Structure
//===== By: ==================================================
//= rAthena Dev Team
//===== Last Updated: ========================================
//= 20210624
//===== Description: =========================================
//= Explanation of the mob_item_ratio.yml file and structure.
//============================================================

> ItemID base drop rates defined in mob_db will not get multiplied by global item_rate* values (aka drop rates) from
> conf/battle/drops.conf. Instead Ratio will be used (100 = 1x).
> If no MonsterID is specified, all monsters will be affected, otherwise only listed ones.

Reviewer: ```suggestion
Item base drop rates defined in mob_db.yml will not get multiplied by global item_rate* values (aka drop rates) from
conf/battle/drops.conf and instead, Ratio will be used (100 = 1x).
If no Mob is specified, all monsters will be affected, otherwise only the ones listed.
```
